indico/modules/events/surveys/templates/management/survey_actions/active.html

```diff
@@ -29,10 +29,24 @@
     </div>
     {% block now_button %}
         <div class="group">
-            <a href="#" data-href="{{ url_for('.close_survey', survey) }}" data-method="POST"
-               class="i-button highlight">
-                {% trans %}Close now{% endtrans %}
+            <a href="#" class="i-button icon-close arrow js-dropdown" data-toggle="dropdown">
+                {% trans -%}Close the survey{%- endtrans %}
             </a>
+            <ul class="dropdown">
+                <li>
+                    <a href="#" data-href="{{ url_for('.close_survey', survey) }}" data-method="POST">
+                        {% trans %}Close now{% endtrans %}
+                    </a>
+                </li>
+                <li>
+                    {% if survey.is_scheduled %}
+                        <a href="#" data-method="POST" data-href="{{ url_for('.unschedule_survey', survey) }}"
+                           data-title="{% trans %}Unschedule survey{% endtrans %}">
```

Comment:
Is this used by anything right now or did you simply forget to remove it when removing the confirmation prompt?